packages/angular_devkit/build_angular/src/utils/i18n-options.ts

```diff
@@ -127,6 +130,13 @@ export async function configureI18nBuild<T extends BrowserBuilderSchema | Server
   }
 
   if (i18n.inlineLocales.size > 0) {
+    const localeDataBasePath = findLocaleBaseDataPath();
```

Comment:
Should one of `localeDataBasePath` or `findLocaleBaseDataPath` change to be consistent with each other?

Reply:
yes. definitely.